### ISSUE TITLE
Named export 'Mask' not found.

### ISSUE THREAD

**Issue by lolcabanon** [ionited/mask#2]
Same thing as [https://github.com/ionited/mask-svelte/issues/1](https://github.com/ionited/mask-svelte/issues/1).

The problem seems to be gone with `@ionited/mask-svelte` but not `@ionited/mask`.

```
import { Mask } from '@ionited/mask';
```

```
[vite] Named export 'Mask' not found.
The requested module '@ionited/mask' is a CommonJS module, which may not support all module.exports as named exports. 

CommonJS modules can always be imported via the default export, for example using

 import pkg from '@ionited/mask';
const {Mask} = pkg;
```

The workaround (`import pkg from '@ionited/mask'; const {Mask} = pkg;`) <strike>**works this time**. Up to you to decide if this need a "fix" or not.</strike>

**Edit :**
Looks like I've written too fast, `import pkg from '@ionited/mask'; const {Mask} = pkg;` **still does not work**!
Only `import * as Mask from '@ionited/mask';` works.

---

| Pacakage | Version |
| -- | -- |
| @ionited/mask | 0.6.1 |
| @ionited/mask-svelte | 0.2.1 |
| node | 20.11.1 |
| pnpm | 8.15.5 |
| svelte | 4.2.12 |
| @sveltejs/kit | 2.5.5 |
| vite | 5.2.7 |
| typescript | 5.4.3 |

**Comment by gabrielcursino** [comment]
@lolcabanon fixed with [0.6.2](https://github.com/ionited/mask/releases/tag/0.6.2) release!

**Comment by lolcabanon** [comment]
![Screenshot_20240409-234343.jpg](https://github.com/ionited/mask/assets/90099349/4d5a2212-7b57-470c-af97-f3e1df6ca521)

Awesome!

I tried many JS / Svelte masking libs and `@ionated/mask` is now officially part of my toolbox.

Thanks for your work!
